css/selectors/part.json

```diff
@@ -0,0 +1,71 @@
+{
+  "css": {
+    "selectors": {
+      "part": {
+        "__compat": {
+          "description": "<code>::part</code>",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/::part",
+          "support": {
+            "chrome": {
+              "version_added": "73"
+            },
+            "chrome_android": {
+              "version_added": "73"
+            },
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "69",
+              "flags": [
+                {
+                  "name": "layout.css.shadow-parts.enabled",
+                  "type": "preference",
+                  "value_to_set": "true"
+                }
+              ]
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": null
+            },
+            "qq_android": {
+              "version_added": false
```

Comment:
I don't really think that we need to specify QQ and UC in here, especially since they're based off of Chromium (plus we've never really worried about them much).  Not really a big deal, just my personal opinion!